packages/asset-buyer/src/types.ts

```diff
@@ -117,7 +124,19 @@ export enum AssetBuyerError {
     TransactionValueTooLow = 'TRANSACTION_VALUE_TOO_LOW',
 }
 
+/**
+ * orders: An array of signed orders
+ * remainingFillableMakerAssetAmounts: A list of fillable amounts for the signed orders. The index of an item in the array associates the amount with the corresponding order.
+ */
 export interface OrdersAndFillableAmounts {
     orders: SignedOrder[];
     remainingFillableMakerAssetAmounts: BigNumber[];
 }
+
+/**
+ * Represents available liquidity for a given assetData
+ */
+export interface LiquidityForAssetData {
+    tokensAvailableInBaseUnits: BigNumber;
```

Comment:
did we decide if we are gonna prepend bigNumber here? we don't have a precedent for it but it might be helpful

Reply:
per convo, every # in AssetBuyer is already a BigNumber so no on this, only wanted to append bigNumber to the var name if it was exposed on the global `zeroExInstant` object